lua/ttt2/libraries/loadingscreen.lua

```diff
@@ -0,0 +1,195 @@
+---
+-- Handles the loading screen that is shown on map reload.
+-- @author Mineotopia
+
+if SERVER then
+    AddCSLuaFile()
+
+    util.AddNetworkString("TTT2LoadingscreenStart")
+end
+
+loadingscreen = loadingscreen or {}
+
+loadingscreen.isInReloading = false
+loadingscreen.wasInReloading = false
```

Comment:
Can we maybe rename this to something like `isShown` and `wasShown`, because reloading is rather specific to the round reload use case and this way it is much clearer what those states mean.